enhancements/update/upgrade-without-registry.md

```diff
@@ -0,0 +1,321 @@
+---
+title: Upgrade without registry
+authors:
+- "@jhernand"
+reviewers:
+- "@avishayt"
+- "@danielerez"
+- "@mrunalp"
+- "@nmagnezi"
+- "@oourfali"
+approvers:
+- "@sdodson"
+- "@zaneb"
+creation-date: 2023-06-29
+last-updated: 2023-06-29
+tracking-link: []
+see-also:
+- https://issues.redhat.com/browse/OCPBUGS-13219
+- https://github.com/openshift/cluster-network-operator/pull/1803
+replaces: []
+superseded-by: []
+---
+
+# Upgrade without registry
+
+## Summary
+
+Provide a documented and supported mechanism to upgrade a cluster without requiring an image
+registry server.
+
+## Motivation
+
+All these stories are in the context of disconnected clusters with limited resources, both in the
+cluster itself and in the surrounding environment:
+
+- The cluster is not connected to the Internet.
+- It isn't possible to bring up additional machines, even temporarily.
+- The resources of the cluster are limited, in particular in terms of CPU and memory.
+
+These clusters are usually installed at the customer site by a partner engineer collaborating with a
+customer technician.
+
+Eventually, the cluster will need to be upgraded, and then the technician will need a supported and
+documented procedure to do it.
+
+### User stories
+
+#### Prepare an upgrade package
+
+As an engineer working in a partner's factory, I want to be able to assemble the upgrade package
+that will be delivered to the technicians that will perform the upgrade on-site.
+
+#### Upgrade a single-node cluster
+
+As a technician, I want to be able to upgrade a single node cluster using the upgrade package and
+without requiring/deploying an image registry server neither in the cluster itself nor externally.
+
+#### Upgrade a multi-node cluster
+
+As a technician, I want to be able to upgrade a multi-node cluster using the upgrade package and
+without requiring/deploying an image registry server neither on the cluster itself nor externally.
+
+### Goals
+
+Provide a documented and supported mechanism that technicians can use to upgrade a cluster without
+requiring a registry server. This upgrade will include the OpenShift core components as well as
+additional applications provided by the partner.
```

Comment:
For the sake of this enhancement I think it is enough to mention only the OpenShift core components.